src/loci/formats/in/ZarrReader.java

```diff
@@ -82,17 +88,26 @@
 
 public class ZarrReader extends FormatReader {
 
+  public static final String QUICK_READ_KEY = "zarrreader.quick_read";
```

Comment:
Minor but longer term consideration of "zarrreader." vs "omezarrreader." vs "omezarr." as a prefix.